### ISSUE TITLE
[Bug]: 결과물에 ad가 적용되지 않습니다.

### ISSUE THREAD

**Issue by Neytiri7** [Bing-su/adetailer#283]
### Describe the bug

그 어떤 오류 메시지도 안뜨고,
Web UI에서도 제대로 동작하는것 처럼 이미지가 확대 되고 CMD에서도 동작하는게 보이는데...

결과물에서 적용이 안됩니다.

오류 메시지나 문제가 보이지도 않아서...

ad를 지우고 다시 설치해도 동일합니다.
master, dev 모두다 동일하게 적용이 안됩니다.

### Screenshots

https://arca.live/b/aiartreal/84078740
https://arca.live/b/aiartreal/84071391
https://arca.live/b/aiartreal/83990476

### Console logs, from start to end.

```Shell
파이썬 3.10.11
버전 1.5.1
토치 : 2.01_CU118
X포머 : 0.0.20
그라디오 : 3.39.0
체크포인트 : F7EEC78FD7
```

**Comment by Bing-su** [comment]
에러가 있던 없던 콘솔 로그 전체는 올려주십시오

**Comment by Bing-su** [comment]
콘솔로그 **전체**에서 전체는 앞부분이나 뒷부분, 혹은 중간 어디든 잘라내거나 수정하지 않고, 어쩌면 몇백줄이 될 수도 있는 원본 그대로를 말합니다.

**Comment by Neytiri7** [comment]
해당 이슈가 재현되는 부분이 아닌, CMD 콘솔창 전부로 수정했습니다.

**Comment by Neytiri7** [comment]
master 브랜치 콘솔입니다.


Already up to date.
venv "O:\AI\SynologyDrive\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.5.1
Commit hash: 68f336bd994bed5442ad95bad6b6ad5564a5409a






Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\a1111-sd-webui-lycoris':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\a1111-sd-webui-tagcomplete':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\adetailer':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\loopback_scaler':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\lora-prompt-tool':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\multidiffusion-upscaler-for-automatic1111':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\posex':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-CPU-GPU-temp-prot':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-dynamic-thresholding':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-agent-scheduler':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-ar':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-bmab':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-color-enhance':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-controlnet':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-ddsd':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-model-converter':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-openpose-editor':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-regional-prompter':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-supermerger':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sdweb-easy-prompt-selector':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\stable-diffusion-webui-images-browser':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\stable-diffusion-webui-localization-ko_KR':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\stable-diffusion-webui-wildcards':
Already up to date.

Pulled changes for repository in 'O:\AI\SynologyDrive\stable-diffusion-webui\extensions\ultimate-upscale-for-automatic1111':
Already up to date.

Launching Web UI with arguments: --opt-split-attention --autolaunch --update-all-extensions --no-half-vae --enable-insecure-extension-access --lowvram --xformers --xformers-flash-attention --opt-channelslast
Tag Autocomplete: Could not locate model-keyword extension, Lora trigger word completion will be limited to those added through the extra networks menu.
[-] ADetailer initialized. version: 23.8.0, num models: 9
[lora-prompt-tool] Get Custom Model Folder
Installing None
2023-08-19 18:05:25,223 - ControlNet - INFO - ControlNet v1.1.238
ControlNet preprocessor location: O:\AI\SynologyDrive\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2023-08-19 18:05:25,357 - ControlNet - INFO - ControlNet v1.1.238
Loading weights [5bc5a45624] from O:\AI\SynologyDrive\stable-diffusion-webui\models\Stable-diffusion\BracingEvoMix_v1.safetensors
Creating model from config: O:\AI\SynologyDrive\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: O:\AI\SynologyDrive\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.ckpt
Applying attention optimization: xformers... done.
Model loaded in 4.2s (load weights from disk: 0.4s, create model: 0.5s, apply weights to model: 0.6s, apply channels_last: 0.5s, apply half(): 0.5s, load VAE: 0.3s, load textual inversion embeddings: 0.3s, calculate empty prompt: 0.9s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
INFO - [AgentScheduler] Task queue is empty
INFO - [AgentScheduler] Registering APIs
Startup time: 72.5s (launcher: 36.5s, import torch: 6.2s, import gradio: 1.5s, setup paths: 1.3s, other imports: 1.4s, setup codeformer: 0.1s, load scripts: 11.5s, scripts before_ui_callback: 0.5s, create ui: 11.3s, gradio launch: 0.6s, app_started_callback: 1.3s).
                                  Calculating sha256 for O:\AI\SynologyDrive\stable-diffusion-webui\models\Stable-diffusion\Neytiri-3.Comic-v1.0.safetensors: 08f6c71d1d985ef2d289bacb0e371076db2c57d36b46c4db28f74e905e7fce49
Loading weights [08f6c71d1d] from O:\AI\SynologyDrive\stable-diffusion-webui\models\Stable-diffusion\Neytiri-3.Comic-v1.0.safetensors
Loading VAE weights specified in settings: O:\AI\SynologyDrive\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.ckpt
Applying attention optimization: xformers... done.
Weights loaded in 4.2s (calculate hash: 2.9s, load weights from disk: 0.4s, apply weights to model: 0.6s, load VAE: 0.4s).
File O:\AI\SynologyDrive\stable-diffusion-webui\extensions\stable-diffusion-webui-wildcards\wildcards\,
.txt not found for the __,
__ wildcard.
File O:\AI\SynologyDrive\stable-diffusion-webui\extensions\stable-diffusion-webui-wildcards\wildcards\|.txt not found for the __|__ wildcard.
100%|██████████████████████████████████████████████████████████████████████████████████| 73/73 [03:41<00:00,  3.04s/it]
ScuNET tiles: 100%|████████████████████████████████████████████████████████████████████| 12/12 [00:01<00:00,  7.58it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 23/23 [01:52<00:00,  4.88s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 96/96 [07:39<00:00,  4.88s/it]
0: 640x448 1 face, 115.4ms
Speed: 7.5ms preprocess, 115.4ms inference, 6.6ms postprocess per image at shape (1, 3, 640, 448)
        Tile 1/6
        Tile 2/6
        Tile 3/6
        Tile 4/6
        Tile 5/6
        Tile 6/6
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [01:30<00:00,  3.01s/it]

0: 640x448 2 hands, 79.5ms
Speed: 2.5ms preprocess, 79.5ms inference, 0.5ms postprocess per image at shape (1, 3, 640, 448)
        Tile 1/2
        Tile 2/2
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [01:30<00:00,  3.02s/it]
        Tile 1/2
        Tile 2/2
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [01:30<00:00,  3.02s/it]

0: 640x448 1 person, 134.5ms
Speed: 3.0ms preprocess, 134.5ms inference, 3.7ms postprocess per image at shape (1, 3, 640, 448)
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [01:31<00:00,  3.04s/it]
        Tile 1/15
        Tile 2/15
        Tile 3/15
        Tile 4/15
        Tile 5/15
        Tile 6/15
        Tile 7/15
        Tile 8/15
        Tile 9/15
        Tile 10/15
        Tile 11/15
        Tile 12/15
        Tile 13/15
        Tile 14/15
        Tile 15/15
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
mediapipe: 1 detected.
        Tile 1/4
        Tile 2/4
        Tile 3/4
        Tile 4/4
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [01:33<00:00,  3.11s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 96/96 [19:34<00:00, 12.23s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 96/96 [19:34<00:00,  4.88s/it]

**Comment by Bing-su** [comment]
on - off 결과물이 다른 걸 보면, 정상작동하고 있는것같고, 이미지가 이상한 건 설정 문제같습니다.

**Comment by Neytiri7** [comment]
설정을 따로 건든건 없는데... 일단, 설정을 초기화 한 다음에 다시 세팅해서, 진행 한번 해보고 다시 말씀 드리겠습니다.

**Comment by Neytiri7** [comment]
여러번 재설치 해보고, 여러번 스탭을 밟아서, 재현 해본결과, 
person_yolo8n-seg.pt 사용시 이 모델이 마지막에 적용되면서,
정상으로 적용된 얼굴 이미지가 이상하게 변형됩니다.

첫번째에 두고, 나머지 모델을 적용하면 이미지가 정상적으로 나오네요.

AD에 사용되는 모델들을 다시 한번 확인 부탁드립니다.
